lib/Red/Driver/Pg.pm6

```diff
@@ -34,6 +34,22 @@ multi method translate(Red::AST::Select $_, $context?, :$gambi where !*.defined)
     self.Red::Driver::CommonSQL::translate($_, $context, :gambi);
 }
 
+multi method translate(Red::AST::In $_, $context?) {
+    if .right.value ~~ Positional {
+        my ($lstr, @lbind) := do given self.translate: .left,  $context { .key, .value }
+
+        if .right.value.elems == 0 {
+            return "$lstr { .op } (SELECT 0 WHERE false)" => @lbind;
+        }
+
+        my $in-placeholder = '(' ~ (self.wildcard xx .right.value.elems).join(',') ~ ')';
```

Comment:
If you fix one of my errors (https://github.com/FCO/Red/blob/master/lib/Red/Driver/CommonSQL.pm6#L521) changing `"?"` to `self.wildcard`, you could just use the `self.translate: .right, $context`